corehq/apps/users/static/users/js/mobile_workers.js

```diff
@@ -321,6 +320,40 @@ hqDefine("users/js/mobile_workers",[
             return self.STATUS.SUCCESS;
         });
 
+        self.passwordSatisfyLength = ko.computed(function () {
+            if (!self.stagedUser()) {
+                return true;
+            }
+
+            if (self.stagedUser().force_account_confirmation()) {
+                return true;
+            }
+
+            if (self.stagedUser().force_account_confirmation_by_sms()) {
+                return true;
+            }
+
+            if (!self.useStrongPasswords()) {
+                // No validation
+                return true;
+            }
+
+            if (!self.skipStandardValidations()) {
+                var minimumPasswordLength = initialPageData.get('minimumPasswordLength');
+                var password = self.stagedUser().password();
+                if (!password) {
+                    return true;
+                }
+                if (self.isSuggestedPassword()) {
+                    return true;
+                }
```

Comment:
Is this check necessary? If we're suggesting passwords that are too short, I think that's a bug we need to fix, rather than just pass through.

Reply:
You're so right!